Sources/TuistLoader/Models+ManifestMappers/Plugin+ManifestMapper.swift

```diff
@@ -6,7 +6,10 @@ import TuistGraph
 import TuistSupport
 
 extension TuistGraph.Plugin {
-    static func from(manifest: ProjectDescription.Plugin) throws -> Self {
+    static func from(
+        manifest: ProjectDescription.Plugin,
+        path _: AbsolutePath
```

Comment:
the path is not needed as it seems ... ?

Reply:
Looks the file is no longer needed 👍🏼